src/Corrade/Utility/Debug.cpp

```diff
@@ -42,6 +49,46 @@ template<> inline void toStream<Implementation::DebugOstreamFallback>(std::ostre
     value.apply(s);
 }
 
+#ifdef CORRADE_TARGET_WINDOWS
+HANDLE getStdHandle(const std::ostream* s) {
+    if(!s) return INVALID_HANDLE_VALUE;
+
+    return s == &std::cout ? ::GetStdHandle(STD_OUTPUT_HANDLE)
+         : s == &std::cerr ? ::GetStdHandle(STD_ERROR_HANDLE)
+         : INVALID_HANDLE_VALUE;
+}
+
+template<Debug::Color c> WORD mapConsoleColor() {
+    switch(c) {
+        case Debug::Color::Black:   return 0;
+        case Debug::Color::Red:     return FOREGROUND_RED;
+        case Debug::Color::Green:   return FOREGROUND_GREEN;
+        case Debug::Color::Yellow:  return FOREGROUND_RED|FOREGROUND_GREEN;
+        case Debug::Color::Blue:    return FOREGROUND_BLUE;
+        case Debug::Color::Magenta: return FOREGROUND_RED|FOREGROUND_BLUE;
+        case Debug::Color::Cyan:    return FOREGROUND_GREEN|FOREGROUND_BLUE;
+        case Debug::Color::White:   return FOREGROUND_RED|FOREGROUND_GREEN|FOREGROUND_BLUE;
+        case Debug::Color::Default: return 0;
```

Comment:
This is not really a runtime switch. `c` is a template parameter and the compiler can optimize this switch easily to a single `return` statement.

I don't like copy&pasting from platform headers if not _really_ required.